imports/components/forms/Checkbox.js

```diff
@@ -81,12 +74,11 @@ export default class Checkbox extends React.Component {
         </h6>
 
         <input
-            ref={this.props.id || this.props.label || this.props.name}
             id={this.props.id || this.props.label || this.props.name}
             type={this.props.type || "checkbox"}
             name={this.props.name || this.props.label}
             className={this.props.inputClasses}
-            disabled={this.disabled()}
+            disabled={this.props.disabled ? true : false}
```

Comment:
Oh i guess this handles the case of `this.props.disabled` being undefined. Carry on. I still think we should have prop types though